Astro CLI v1.0.0 (All Changes) (#690)
astronomer/docs: software/environment-variables.md

### via `.env` (_Local Only_)

> The [Astro CLI](cli-quickstart.md) comes with the ability to bring in Environment Variables from a specified `.env` file, which was automatically generated when you initialized an Airflow project on Astronomer via `$ astro dev init`.

Reviewer: ```suggestion
You can use the [Astro CLI](cli-quickstart.md) to import Environment Variables from the `.env` file, that was automatically generated when you initialized an Airflow project on Astronomer using`$ astro dev init`.
```
